### ISSUE TITLE
Expand container

### ISSUE THREAD

**Issue by Shadowfaxenator** [goinstant/webrtc#17]
I add this to options:
`expandContainer: document.getElementById('myExpandContainer'),`

This tag is added to index.html
`<div id="myExpandContainer"></div>`

Widget works, but if I push expand button on a user tag, it does not expand it and doesn't add styles, it only moves it to `myExpandContainer` but no expanding. I saw CSS file and there are classes for this.

**Comment by colinmacdonald** [comment]
Yes, we left the styles up to the developer when using the `expandContainer` option as they may want to change various things when a user is expanded.
